dev/dvm/bin/dvm

```diff
@@ -11,6 +11,6 @@ require "dvm"
 
 begin
   DVM::Application.start(ARGV)
-rescue DVM::DVMArgumentError => e
```

Comment:
It would/should have dumped a stack if it wasn't an error specifically thrown by dvm internals. (I overused the crap out of DVMArgumentError ).  

@irvingpop is this one still needed after the re-addition of the missing DVMArgumentError class?  Generally dvmargumenterrors are things that will have a hint as to how to fix (now that they actually exist again), while the general Exception case we'd want to see the stack trace to figure out where it broke.   

Or am I misremembering the default behavior of the thor app when it hits an unhandled exception?